Core/OxidEeEvents.php

```diff
@@ -361,6 +353,25 @@ public static function onDeactivate()
     {
         self::$_oDb = DatabaseProvider::getDb();
         self::_disablePaymentMethods();
+        self::_clearCache();
+    }
+
+    /**
+     * Clears cache
+     *
+     * @since 1.2.0
+     */
+    private static function _clearCache()
```

Comment:
This clears the _file_ cache only, so it should probably be named accordingly.

Reply:
👍